src/test/run-make-fulldeps/split-dwarf/Makefile

```diff
@@ -3,6 +3,6 @@
 # only-linux
 
 all:
-	$(RUSTC) -Z split-dwarf=split foo.rs
+	$(RUSTC) -Z split-dwarf=split -C debuginfo=2 foo.rs
```

Comment:
`-Z split-dwarf=split` no longer automatically implies that debuginfo is enabled. This is in line with changes in GCC and clang (https://reviews.llvm.org/D80391), though I'm not sure what changed to affect rustc (the referenced revision only affects the frontend).